require/require.go

```diff
@@ -33,6 +40,20 @@ func Contains(t TestingT, s interface{}, contains interface{}, msgAndArgs ...int
 	}
 }
 
+// Containsf asserts that the specified string, list(array, slice...) or map contains the
+// specified substring or element.
+//
+//    assert.Containsf(t, "Hello World", "World", "extra error message")
```

Comment:
This is part of the code-generated comments.

`, "extra error message"` is added to the end of usage examples on "formatted assertions".

Would need to figure out what we can add instead that would be a relevant non-confusing message+variable that works for any assertion.